api/v1alpha1/webhook_suite_test.go

```diff
@@ -31,7 +31,7 @@ import (
 	admissionv1beta1 "k8s.io/api/admission/v1beta1"
 	//+kubebuilder:scaffold:imports
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/rest"
+	// "k8s.io/client-go/rest"
```

Comment:
can it be removed?

Reply:
I'll reply once for all 4 commented lines. I commented instead of deleting them because this is generated code, so I assume we will need it once we implement the test. But, if you think it's better to remove them, I don't mind

Reply:
So let's remove it and maybe open a relevant task in Jira for implementing those tests

Reply:
Done